lesson2/argirovatina_homework_2.rb

```diff
@@ -0,0 +1,35 @@
+module M
+
+  def self.included(base)
+    base.extend(C)
+  end
+
+  def custom_instance_method
+    p Foo.instance_methods - Object.instance_methods
+  end
+
+  module C
+    def custom_class_method
+      p Foo.methods - Object.methods
+    end
+  end
+
+end
+
+class Foo
+  include M
+end
+
+##############
+
+f = Foo.new
+b = Foo.new
+
+f.custom_instance_method
```

Comment:
Please, add as a comment expected result if you want to perform some method.
Like `object.some_method # => 'my result'`.
But I our case you should not do this, we don't care about a behavior of our methods, that's not the point of the task.